public/lesswrong-editor/src/ckeditor5-mention/theme/mention.css

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
oh, this is empty, can probs get rid of it too! (make sure to remove the import)